### DISCUSSION TITLE
Deletion of account

### DISCUSSION THREAD

**Discussion by johnstagl** [community/community#54478] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Select Feature Area

- [ ] Issues
- [ ] Projects

### Body

We recently moved our system to AWS and we were using Forge, Linode and Cloudflare. I was told by my staff tat we no longer need Linode and that I should delete it (along with the others). When I went to login to Linode, I was prompted to login using GitHub.  Like other platforms, I can login using Google, Facebook etc. and when I do this I am in the system I logged into regardless of the means to login. In this case I logged into Linode using Github, went to the appropriate section to close the account (in Linode) and when I said delete this account, it asked "are you sure you want to delete this aacount" if yes, I had to type I want to delete this.  Not "repository" or "GitHub account"...just "account". I deleted it and found out that it was NOT Linode but it was my GitHub repositories; all of them. There we no distinctions made like, "you are about to delete the following repositories" or "you are about to delete your entire GitHub account".  No it was just "account" when all along I thought I had logged into Linode.  OK, I am stupid for doing this and I am the sole blame but what I do, is design systems and THIS was a design error.  Think about it.  I lost everything when I attempted to close my Linode account after using GitHub to login. Please consider revamping this process.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @johnstagl ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by johnstagl** [comment]
Thank you very much.  I am not that familiar with GitHub as you can tell
from the content of my post.

Nevertheless, I do appreciate your help.

Regards,
John

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
